Week-5/Exercise_5.2/vowel.js

```diff
@@ -0,0 +1,28 @@
+// to cocunt vowels in string
+function vowelCount(str) {
+  // check is it string
+  if (typeof str === "string") {
+    const vMap = new Map();
+    for (const ele of str) {
+      let formatedText = ele.toLowerCase();
+      if (isVowel(formatedText)) {
```

Comment:
this check is not required, instead we can initialize the map with default values as {[vowel]: 0} and we can check if map.has(char) so that Array includes can be avoided